stripe/deploy.gradle

```diff
@@ -0,0 +1,99 @@
+apply plugin: 'maven'
```

Comment:
@mrmcduff-stripe If this is mostly copied in from chrisbanes' repo, can we put a comment with attribute and additional information about that here? It might be especially useful as a pointer in case this stops working down the line and an update upstream is available.